2020-05-13 00:13:49,821 ERROR 9858 [-/127.0.0.1/-/0ms GET /] nodejs.ETIMEDOUTError: connect ETIMEDOUT
    at PoolConnection.Connection._handleConnectTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/mysql/lib/Connection.js:409:13)
    at Object.onceWrapper (events.js:417:28)
    at Socket.emit (events.js:311:20)
    at Socket._onTimeout (net.js:478:8)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
    --------------------
    at Protocol._enqueue (/Users/jorky/code/TexasPokerGame/server/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/jorky/code/TexasPokerGame/server/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/jorky/code/TexasPokerGame/server/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/jorky/code/TexasPokerGame/server/node_modules/mysql/lib/Pool.js:48:16)
    at /Users/jorky/code/TexasPokerGame/server/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/jorky/code/TexasPokerGame/server/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as getConnection] (/Users/jorky/code/TexasPokerGame/server/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at Pool.query (/Users/jorky/code/TexasPokerGame/server/node_modules/mysql/lib/Pool.js:202:8)
    at /Users/jorky/code/TexasPokerGame/server/node_modules/ali-rds/node_modules/pify/index.js:29:7
    sql: select now() as currentTime;
errorno: "ETIMEDOUT"
code: "ETIMEDOUT"
syscall: "connect"
fatal: true
name: "ETIMEDOUTError"
pid: 9858
hostname: 192.168.0.110

2020-05-13 00:25:09,314 ERROR 10142 [-/192.168.0.102/-/22ms GET /socket.io/?room=194379&token=undefined&EIO=3&transport=websocket] room service tick null
2020-05-13 00:55:18,646 ERROR 10142 [-/192.168.0.110/-/17ms GET /socket.io/?room=194379&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Im5pY2tOYW1lIjoiYyIsImFjY291bnQiOiJjYWkiLCJ1c2VySWQiOjF9LCJpYXQiOjE1ODkyODg0OTMsImV4cCI6MTU4OTMxMDA5M30.PXUIHiGCM4BQICZ_BymL0EMaBcsBdXkphPdb93K93Ks&EIO=3&transport=websocket] room service tick null
2020-05-13 00:55:24,222 ERROR 10142 [-/192.168.0.110/-/3ms GET /socket.io/?room=194379&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Im5pY2tOYW1lIjoiYyIsImFjY291bnQiOiJjYWkiLCJ1c2VySWQiOjF9LCJpYXQiOjE1ODkyODg0OTMsImV4cCI6MTU4OTMxMDA5M30.PXUIHiGCM4BQICZ_BymL0EMaBcsBdXkphPdb93K93Ks&EIO=3&transport=websocket] room service tick null
2020-05-13 00:55:27,522 ERROR 10142 [-/192.168.0.102/-/5ms GET /socket.io/?room=194379&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Im5pY2tOYW1lIjoi6JShIiwiYWNjb3VudCI6ImNhaTExMSIsInVzZXJJZCI6Mn0sImlhdCI6MTU4OTMwMDgxNywiZXhwIjoxNTg5MzIyNDE3fQ.yL0K89W1xPF6MnGDpv35IsM8FzxvG4R4d_pwgK0KLDs&EIO=3&transport=websocket] room service tick null
2020-05-13 00:57:37,592 ERROR 10142 [-/192.168.0.110/-/6ms GET /socket.io/?room=194379&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Im5pY2tOYW1lIjoi6JShMSIsImFjY291bnQiOiJjYWkxMSIsInVzZXJJZCI6M30sImlhdCI6MTU4OTI5Mzg4MCwiZXhwIjoxNTg5MzE1NDgwfQ.NDb5EoN-mhLXTPM74_d6rVek5mWAYFVV2QBX6qisuH0&EIO=3&transport=websocket] room service tick null
2020-05-13 00:58:07,733 ERROR 10142 [-/192.168.0.102/-/3ms GET /socket.io/?room=194379&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Im5pY2tOYW1lIjoi6JShIiwiYWNjb3VudCI6ImNhaTExMSIsInVzZXJJZCI6Mn0sImlhdCI6MTU4OTMwMDgxNywiZXhwIjoxNTg5MzIyNDE3fQ.yL0K89W1xPF6MnGDpv35IsM8FzxvG4R4d_pwgK0KLDs&EIO=3&transport=websocket] room service tick null
2020-05-13 02:05:18,858 ERROR 9389 nodejs.ClusterClientNoResponseError: client no response in 3612385ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 9389
hostname: 192.168.0.110

2020-05-13 04:05:27,832 ERROR 9389 nodejs.ClusterClientNoResponseError: client no response in 7208681ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 9389
hostname: 192.168.0.110

2020-05-13 04:05:32,288 ERROR 10142 [-/192.168.0.110/-/20ms GET /socket.io/?room=121127&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Im5pY2tOYW1lIjoi6JShMSIsImFjY291bnQiOiJjYWkxMSIsInVzZXJJZCI6M30sImlhdCI6MTU4OTMwMjY2NywiZXhwIjoxNTg5MzI0MjY3fQ.rwlDPpbPM7t8UXBt8blI_wlibyBxqTxWrhoz9DhQYzg&EIO=3&transport=websocket] room service tick null
2020-05-13 06:05:47,981 ERROR 9389 nodejs.ClusterClientNoResponseError: client no response in 7219167ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 9389
hostname: 192.168.0.110

2020-05-13 08:06:26,296 ERROR 9389 nodejs.ClusterClientNoResponseError: client no response in 7216708ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 9389
hostname: 192.168.0.110

2020-05-13 10:06:43,794 ERROR 9389 nodejs.ClusterClientNoResponseError: client no response in 7217444ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 9389
hostname: 192.168.0.110

2020-05-13 12:07:01,699 ERROR 9389 nodejs.ClusterClientNoResponseError: client no response in 7217818ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 9389
hostname: 192.168.0.110

2020-05-13 13:06:12,446 ERROR 9389 nodejs.ClusterClientNoResponseError: client no response in 3550726ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 9389
hostname: 192.168.0.110

2020-05-13 20:15:12,162 ERROR 9389 nodejs.ClusterClientNoResponseError: client no response in 25739682ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 9389
hostname: 192.168.0.110

2020-05-13 20:15:13,242 ERROR 10142 [-/192.168.0.110/-/34ms GET /socket.io/?room=700435&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Im5pY2tOYW1lIjoiYyIsImFjY291bnQiOiJjYWkiLCJ1c2VySWQiOjF9LCJpYXQiOjE1ODkzMDI2NDYsImV4cCI6MTU4OTMyNDI0Nn0.t01MJ-AhyISF29qJwJ633fKJhwtjpYRAH2-kWf4RFgk&EIO=3&transport=websocket] room service tick null
2020-05-13 20:15:14,770 ERROR 10142 [-/192.168.0.110/-/6ms GET /socket.io/?room=121127&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Im5pY2tOYW1lIjoi6JShMSIsImFjY291bnQiOiJjYWkxMSIsInVzZXJJZCI6M30sImlhdCI6MTU4OTMwMjY2NywiZXhwIjoxNTg5MzI0MjY3fQ.rwlDPpbPM7t8UXBt8blI_wlibyBxqTxWrhoz9DhQYzg&EIO=3&transport=websocket] room service tick null
2020-05-13 21:16:04,054 ERROR 9389 nodejs.ClusterClientNoResponseError: client no response in 3631874ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 9389
hostname: 192.168.0.110

2020-05-13 21:16:48,913 ERROR 9389 nodejs.unhandledExceptionError: ENOENT: no such file or directory, stat '/Users/jorky/code/TexasPokerGame/server/src/app.ts'
    at Object.statSync (fs.js:932:3)
    at DevelopmentEventSource._onFsWatchChange (/Users/jorky/code/TexasPokerGame/server/node_modules/egg-watcher/lib/event-sources/development.js:82:16)
    at FSWatcher.emit (events.js:311:20)
    at FSEvent.FSWatcher._handle.onchange (internal/fs/watchers.js:135:12)
errno: -2
syscall: "stat"
code: "ENOENT"
path: "/Users/jorky/code/TexasPokerGame/server/src/app.ts"
name: "unhandledExceptionError"
pid: 9389
hostname: 192.168.0.110

2020-05-13 21:17:27,063 ERROR 11881 [-/192.168.0.110/-/123ms GET /socket.io/?room=121127&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Im5pY2tOYW1lIjoi6JShMSIsImFjY291bnQiOiJjYWkxMSIsInVzZXJJZCI6M30sImlhdCI6MTU4OTMwMjY2NywiZXhwIjoxNTg5MzI0MjY3fQ.rwlDPpbPM7t8UXBt8blI_wlibyBxqTxWrhoz9DhQYzg&EIO=3&transport=websocket] room service tick null
2020-05-13 21:19:36,297 ERROR 11973 [-/192.168.0.110/-/31ms GET /socket.io/?room=121127&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Im5pY2tOYW1lIjoi6JShMSIsImFjY291bnQiOiJjYWkxMSIsInVzZXJJZCI6M30sImlhdCI6MTU4OTMwMjY2NywiZXhwIjoxNTg5MzI0MjY3fQ.rwlDPpbPM7t8UXBt8blI_wlibyBxqTxWrhoz9DhQYzg&EIO=3&transport=websocket] room service tick null
2020-05-13 21:58:58,415 ERROR 12177 nodejs.ClusterClientNoResponseError: client no response in 1907292ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 12177
hostname: 192.168.0.110

2020-05-13 22:02:27,916 ERROR 12177 nodejs.ClusterClientNoResponseError: client no response in 168906ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 12177
hostname: 192.168.0.110

2020-05-13 22:24:18,590 ERROR 12178 [-/192.168.0.110/-/10ms GET /socket.io/?room=879571&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Im5pY2tOYW1lIjoiYyIsImFjY291bnQiOiJjYWkiLCJ1c2VySWQiOjF9LCJpYXQiOjE1ODkzNzU5NTYsImV4cCI6MTU4OTM5NzU1Nn0.rjTkkfVRAvFwxkutLExEO1SDOBnymyDKnnte2sgQQgw&EIO=3&transport=websocket] room service tick null
2020-05-13 22:24:26,247 ERROR 12178 [-/192.168.0.110/-/2ms GET /socket.io/?room=879571&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Im5pY2tOYW1lIjoiYyIsImFjY291bnQiOiJjYWkiLCJ1c2VySWQiOjF9LCJpYXQiOjE1ODkzNzU5NTYsImV4cCI6MTU4OTM5NzU1Nn0.rjTkkfVRAvFwxkutLExEO1SDOBnymyDKnnte2sgQQgw&EIO=3&transport=websocket] room service tick null
2020-05-13 22:26:12,241 ERROR 12178 nodejs.TypeError: Cannot read property '0' of undefined (uncaughtException throw 1 times on pid:12178)
    at /Users/jorky/code/TexasPokerGame/server/src/app/io/controller/game.ts:80:69
    at processTicksAndRejections (internal/process/task_queues.js:79:11)

pid: 12178
hostname: 192.168.0.110

2020-05-13 22:43:42,477 ERROR 13106 nodejs.ClusterClientNoResponseError: client no response in 186038ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 13106
hostname: 192.168.0.110

2020-05-13 22:54:34,313 ERROR 13106 nodejs.ClusterClientNoResponseError: client no response in 631798ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 13106
hostname: 192.168.0.110

2020-05-13 23:24:58,774 ERROR 14600 [-/192.168.0.110/-/5ms GET /socket.io/?room=467504&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Im5pY2tOYW1lIjoiYyIsImFjY291bnQiOiJjYWkiLCJ1c2VySWQiOjF9LCJpYXQiOjE1ODkzNzU5NTYsImV4cCI6MTU4OTM5NzU1Nn0.rjTkkfVRAvFwxkutLExEO1SDOBnymyDKnnte2sgQQgw&EIO=3&transport=websocket] room service tick null
2020-05-13 23:25:06,186 ERROR 14709 [-/192.168.0.110/-/8ms GET /socket.io/?room=467504&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Im5pY2tOYW1lIjoi6JShMSIsImFjY291bnQiOiJjYWkxMSIsInVzZXJJZCI6M30sImlhdCI6MTU4OTM3NTk4OCwiZXhwIjoxNTg5Mzk3NTg4fQ.HgTT61q2AYW_PbRajx4fSelOdGUtOH2GEsWhPi1IHAA&EIO=3&transport=websocket] room service tick null
2020-05-13 23:25:13,031 ERROR 14709 [-/192.168.0.110/-/2ms GET /socket.io/?room=467504&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Im5pY2tOYW1lIjoi6JShMSIsImFjY291bnQiOiJjYWkxMSIsInVzZXJJZCI6M30sImlhdCI6MTU4OTM3NTk4OCwiZXhwIjoxNTg5Mzk3NTg4fQ.HgTT61q2AYW_PbRajx4fSelOdGUtOH2GEsWhPi1IHAA&EIO=3&transport=websocket] room service tick null
